vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in

```diff
@@ -1,6 +1,6 @@
 SunnyValley: {
   fingerprints: "/usr/local/etc/pkg/fingerprints/SunnyValley",
-  url: "https://updates.sunnyvalley.io/opnsense/${ABI}/%%PLUGIN_ABI%%/OpenSSL/latest",
+  url: "https://updates.zenarmor.com/opnsense/${ABI}/%%PLUGIN_ABI%%/OpenSSL/latest",
```

Comment:
Theoretically you could also drop “OpenSSL” now while at it?

Reply:
yes your right, I removed OpenSSL path

Reply:
Thanks. Is this going to be ready for release in 23.7.5? Not sure if next week or the week after.

Reply:
We are ready, we are performing final checks.

Reply:
Ok, if something goes wrong let me know else I’ll pick it up as discussed. Thanks for your work 😊